### PR TITLE
feat: rename install_token to installation token

### PR DESCRIPTION
**Pull request by sumo-drosiek**
This is continuation of #969

ToDo:

- support for windows both install_token and installation_token
  - [x] As I understand, we are not comparing installation tokens (previous and current installation) right now, so skipping this point
  - [x] Supporting to run in poershell with both `INSTALLTOKEN` and `INSTALLATIONTOKEN`
- update scripts_tests
  - [x] updated
  - [x] tested

### CODE REVIEW

File: pkg/scripts_test/config.go [hunk 1]
```diff
@@ -17,9 +17,10 @@ type extensions struct {
 }
 
 type sumologicExtension struct {
-	InstallToken string            `yaml:"install_token"`
-	Tags         map[string]string `yaml:"collector_fields"`
-	APIBaseURL   string            `yaml:"api_base_url"`
+	InstallToken      string            `yaml:"install_token,omitempty"`
```
Comment (rnishtala-sumo) [inline pkg/scripts_test/config.go:20]: should we remove the install_token entry?

File: scripts/install.sh [hunk 2]
```diff
@@ -766,8 +766,16 @@ function get_user_config() {
...
+    # extract installation_token and strip quotes
+    # fallback to deprecated install_token
+    grep -m 1 installation_token "${file}" \
+        | sed 's/.*installation_token:[[:blank:]]*//' \
+        | sed 's/[[:blank:]]*$//' \
+        | sed 's/^"//' \
+        | sed "s/^'//" \
+        | sed 's/"$//' \
+        | sed "s/'\$//" \
+    || grep -m 1 install_token "${file}" \
```
Comment (rnishtala-sumo) [inline scripts/install.sh:778]: why do we need two `greps` here and not just `grep` for `installation token` in the file?

Comment (sumo-drosiek) [reply scripts/install.sh:778]: Backward compatibility. We want to support previous installations

File: pkg/scripts_test/config.go (hunk 1 above)
Comment (sumo-drosiek) [reply pkg/scripts_test/config.go:20]: I need it to test backward compatibility

**Review by rnishtala-sumo — APPROVED** [review]

**Pull request merged**